pkgs/tools/misc/trezorencrypt/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "trezorencrypt-${version}";
+  version = "v0.1";
+
+  # Fixes Cgo related build failures (see https://github.com/NixOS/nixpkgs/issues/25959 )
+  hardeningDisable = [ "fortify" ];
+
+  goPackagePath = "github.com/petrkr/trezorencrypt";
+
+  src = fetchFromGitHub {
+    owner  = "petrkr";
+    repo   = "trezorencrypt";
+    rev    = "${version}";
```

Comment:
`rev = "v${version}"`

Reply:
So name of nix's package will not contain that 'v' ?right?

Reply:
correct, nixpkgs has version functions that expect the `0.1` form as well.